multisrc/overrides/madara/resetscans/src/ResetScans.kt

```diff
@@ -0,0 +1,7 @@
+package eu.kanade.tachiyomi.extension.en.resetscans
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+
+class ResetScans : Madara("Reset Scans", "https://reset-scans.us", "en") {
+    override val useNewChapterEndpoint = true
+    override val chapterUrlSelector = ".li__text > a"
+}
```

Comment:
```kotlin
class ResetScans : Madara("Reset Scans", "https://reset-scans.com", "en", java.text.SimpleDateFormat("dd/MM/yyyy", Locale.US)) {
    override val useNewChapterEndpoint = true

    override fun chapterFromElement(element: Element): SChapter {
        val chapter = super.chapterFromElement(element)

        with(element) {
            chapter.date_upload = select("span.chapter-release-date i").firstOrNull()?.text().let { parseChapterDate(it) }
        }

        return chapter
    }
}
```

Here's the old code. Is the date format and upload date selector still useful?

Reply:
They've changed the date format to omit the year. I'm not too versed in the SimpleDateFormat specifiers, but it would be something like "MMM dd". (Apr 28, Dec 25, Jan 08, etc.) I don't think you can parse a date without a year.